starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java

```diff
@@ -56,7 +56,22 @@ public void createTest() {
     Assert.assertTrue(inventory1.getName() == NAME);
     Assert.assertTrue(inventory1.getProductType() == PRODUCT_TYPE);
   }
-
+  /**
+   * Test delete method
+   */
+  @Test
+  public void deleteTest() {
+    Inventory inventory = new Inventory();
+    inventory.setName(NAME);
+    inventory.setId(ID);
+    inventory.setProductType(PRODUCT_TYPE);
+    inventory = this.mongoTemplate.save(inventory);
+    String id = inventory.getId();
+    //isPresent confirms that the Inventory still exists but is only deleted from the inventoryDAO
+    Assert.assertTrue(inventoryDAO.delete(id).isPresent());
+    //Inventory should no longer be in the database meaning searching by id should return null
+    Assert.assertNull(this.mongoTemplate.findById(id, Inventory.class));
```

Comment:
We don't actually need this additional check as the previous one tells us everything we need to know.